algorithms/src/Kokkos_Sort.hpp

```diff
@@ -628,6 +633,24 @@ std::enable_if_t<Kokkos::is_execution_space<ExecutionSpace>::value> sort(
   bin_sort.sort(exec, view);
 }
 
+#if defined(KOKKOS_ENABLE_SYCL)
+template <class DataType, class... Properties>
+void sort(const Experimental::SYCL& space,
+          const Kokkos::View<DataType, Properties...>& view,
+          bool const = false) {
+  using ViewType = Kokkos::View<DataType, Properties...>;
+  static_assert(
+      ViewType::rank == 1 &&
+          !std::is_same<typename ViewType::array_layout, LayoutStride>::value,
```

Comment:
I think the check should be affirmative that the layout is contiguous, rather than excluding one known bad case

Reply:
Done.